src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -2106,6 +2106,12 @@ void Unit::AttackerStateUpdate(Unit* victim, WeaponAttackType attType, bool extr
         DamageInfo dmgInfo(damageInfo);
         ProcSkillsAndAuras(damageInfo.target, damageInfo.procAttacker, damageInfo.procVictim, PROC_SPELL_TYPE_NONE, PROC_SPELL_PHASE_NONE, dmgInfo.GetHitMask(), nullptr, &dmgInfo, nullptr);
 
+        if (dmgInfo.GetHitMask() & PROC_HIT_CRITICAL) // @todo is there a better place to put this hack? there doesn't seem to be any aura applied on talent
+            if (Pet* pet = ToPet())
+                if (pet->HasSpell(53508)) // Wolverine Bite
+                    pet->AddComboPoints(damageInfo.target, 1);
```

Comment:
`TRIGGER_ACTIVATE` is a client side flag that interacts with other flags (in this case `REQ_COMBO_POINTS1`) to make the icon gray out if not available. No server handling for these spells.

NB: Most other spells with this attribute have proc info (f.ex. Art of War).